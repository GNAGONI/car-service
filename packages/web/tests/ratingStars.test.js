import React from 'react'
import renderer from 'react-test-renderer'
import RatingStars from 'components/ratingStars'

it('Renders correctly', () => {
  const tree = renderer.create(<RatingStars />).toJSON()
  expect(tree).toMatchSnapshot()
})
