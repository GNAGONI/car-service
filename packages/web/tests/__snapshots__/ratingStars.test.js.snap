// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<ul
  className="star-rating"
>
  <li
    className=""
  >
    <span
      className="sr-only"
    >
      1 Star
    </span>
  </li>
  <li
    className=""
  >
    <span
      className="sr-only"
    >
      2 Star
    </span>
  </li>
  <li
    className=""
  >
    <span
      className="sr-only"
    >
      3 Star
    </span>
  </li>
  <li
    className=""
  >
    <span
      className="sr-only"
    >
      4 Star
    </span>
  </li>
  <li
    className=""
  >
    <span
      className="sr-only"
    >
      5 Star
    </span>
  </li>
</ul>
`;
